eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/repository/ReminderRepository.java

```diff
@@ -3,10 +3,14 @@
 import hu.flowacademy.companycalendar.model.Reminder;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface ReminderRepository extends JpaRepository<Reminder, Long> {
 
   List<Reminder> findByUserId(Long userid);
+
+  @Query(value = "SELECT * FROM reminder  WHERE user_id = ?1 AND ending_time > ?2", nativeQuery = true)
```

Comment:
Miért native query?

Reply:
Izé ... nem az lesz, de elsőre így egyszerűbb volt. Igazából ezt nem is akartam felpusholni, csak teszt volt magamnak, hogy jó-e a a logika, de nem figyeltem és felment.
JPQL ofkórsz